src/main/resources/assets/twilightforest/lang/de_de.json

```diff
@@ -606,45 +707,45 @@
   "item.twilightforest.knightmetal_axe.tooltip": "Extraschaden bei ungepanzerten Gegnern",
   "item.twilightforest.giant_pickaxe.tooltip": "Zum Abbauen von Riesenblöcken",
   "item.twilightforest.fiery_pickaxe.tooltip": "Automatisches Schmelzen",
-  "item.twilightforest.fiery_sword.tooltip": "Setzt Gegner in Brand",
-  "item.twilightforest.fiery_helmet.tooltip": "Setzt Angreifer in Brand",
-  "item.twilightforest.fiery_chestplate.tooltip": "Setzt Angreifer in Brand",
-  "item.twilightforest.fiery_leggings.tooltip": "Setzt Angreifer in Brand",
-  "item.twilightforest.fiery_boots.tooltip": "Setzt Angreifer in Brand",
-  //"item.twilightforest.phantom_helmet.tooltip": "Is never lost on death",
-  //"item.twilightforest.phantom_chestplate.tooltip": "Is never lost on death",
+  "item.twilightforest.fiery_sword.tooltip": "Setzt den Gegner in Brand",
+  "item.twilightforest.fiery_helmet.tooltip": "Setzt den Angreifer in Brand",
+  "item.twilightforest.fiery_chestplate.tooltip": "Setzt den Angreifer in Brand",
+  "item.twilightforest.fiery_leggings.tooltip": "Setzt den Angreifer in Brand",
+  "item.twilightforest.fiery_boots.tooltip": "Setzt den Angreifer in Brand",
+  "item.twilightforest.phantom_helmet.tooltip": "Nie verloren im Tod",
+  "item.twilightforest.phantom_chestplate.tooltip": "Nie verloren im Tod",
   "item.twilightforest.yeti_helmet.tooltip": "Friert Angreifer ein",
   "item.twilightforest.yeti_chestplate.tooltip": "Friert Angreifer ein",
   "item.twilightforest.yeti_leggings.tooltip": "Friert Angreifer ein",
   "item.twilightforest.yeti_boots.tooltip": "Friert Angreifer ein",
 
-  //"item.twilightforest.arctic_armor.tooltip": "Dyeable like Leather Armor",
+  "item.twilightforest.arctic_armor.tooltip": "Einfärbbar wie Lederrüstungen",
 
-  "itemGroup.twilightforest": "Twilight-Forest",
 
+  // Entitäten
   "entity.twilightforest.wild_boar": "Wildschwein",
   "entity.twilightforest.bighorn_sheep": "Dickhornschaf",
-  "entity.twilightforest.deer": "Wilder Hirsch",
+  "entity.twilightforest.deer": "Hirsch",
```

Comment:
The deer is called "Wild Deer", so im not sure if this change was intended or not